feedback-module/src/lib/hooks/moduleOnScreen.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+
+export default function moduleOnScreen(ref) {
+  const [isIntersecting, setIntersecting] = useState(false);
```

Comment:
Could we move `const [userViewed, setUserViewed] = useState(false);` here, move checkVisible() to this file and change the if statement to `if (!userViewed && isIntersecting)`, and then call `checkVisible()` after line 7? I think we could also combine this hook with the other hook, so that we have easy access to the userViewedModule() method.

Reply:
The only problem with this is wouldn't it reset userViewed to false every time. We only want to send this event once

Reply:
I don't think so— we never call setUserViewed(false) and since we're not actually changing/reloading the page or anything like that, `moduleOnScreen(ref)` only gets called once so the state should stay the same on all the screens; I don't think it's functionally any different than having the `useState(false)` for `userViewed` in Module.js.

Reply:
I'll test it out and see what happens